### ISSUE TITLE
CLI --colors options [never|auto|always] are undocumented

### ISSUE THREAD

**Issue by Jean85** [sebastianbergmann/phpunit-documentation#319]
Recently I was investigating into finding a way to disable colored output from CLI, and the first and foremost Google result about it is this: https://github.com/sebastianbergmann/phpunit/issues/516

Contrary to that (and to documentation), I tried this:

```
bin/phpunit --colors=no
Argument #3 (No Value) of PHPUnit_TextUI_ResultPrinter::__construct() must be a value from "never", "auto" or "always"
```

And it led me to discover what I needed. So, documentation about this option need to be enlarged...

**Comment by henriquemoody** [comment]
That's weird, see #247.

**Comment by Jean85** [comment]
I've investigated further. I'm using 4.8.5 and with `--help` the flags are shown, that's good.

They're missing from the current static docs v4.8: https://phpunit.de/manual/current/en/textui.html#textui.clioptions

The static page refers to a 4.8.0 version, so someone fixed it in the CLI between .0 and .5, and didn't report it to the docs,

**Comment by Jean85** [comment]
Found the culprit: 1505497608880969732f8cd1d46bfd545cc360b0
The docs where present since 4.6 (thanks to your commit c9d3e0fcef21e56ac84a5bfa3a3dd6194b9b9aa2,@henriquemoody) but a subsequent commit removed it.

I'll try to regroup all the missing pieces in a pull request.

**Comment by henriquemoody** [comment]
That would be awesome, also looks like `--columns`'s description was removed as well, can you restore it too?

**Comment by Jean85** [comment]
Just edited my previous message up there, I'm working on it too :+1: 
What about 4.7? It's dropped from support?
I'm investigating the translations too...

**Comment by Jean85** [comment]
Solved by merging #320 ;)
